catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -19,6 +23,52 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
   );
   const [isActive, setIsActive] = useState(false);
 
+  // Gets the image as it might not be in the current results, and displays it.
+  const getImage = async routerImageId => {
+    const apiUrl = `https://api.wellcomecollection.org/catalogue/v2/images/${routerImageId}`;
+    const image: Image = await fetch(apiUrl).then(res => res.json());
+
+    if (image) {
+      setExpandedImage(image);
+      setIsActive(true);
+    }
+  };
+
+  useEffect(() => {
+    // If there is a set expandedImage and it's a different one that the current queried one
+    // Change URL to reflect the change
+    if (!!expandedImage && routerImageId !== expandedImage.id) {
+      router.push(
+        `${router.pathname}?${page ? 'page=' + page + '&' : ''}imageId=${
+          expandedImage.id
+        }`,
+        undefined,
+        {
+          shallow: true,
+        }
+      );
+    }
+  }, [expandedImage]);
+
+  useEffect(() => {
+    // If isActive changes to false, reset the URL to no imageId
+    if (!isActive) {
+      router.push(
+        `${router.pathname}${page ? '?page=' + page : ''}`,
+        undefined,
+        {
+          shallow: true,
+        }
+      );
+    }
+  }, [isActive]);
+
+  useEffect(() => {
+    // If there is an imageId in the query, fetch that image and display it
+    // Otherwise ensure modal is closed and URL gets resetted
```

Comment:
resetted to reset

Reply:
thanks, my English tends to go down when ill 😄  I'll change both and merge!